docs/guides/self-signed-ca-certs.md

```diff
@@ -0,0 +1,50 @@
+---  
+title: Fetch Packages from Registry with Self-Signed CA Certificate  
+toc: true  
+weight: 270  
+indent: true  
+---  
+
+# Overview of Crossplane for Registry with Self-Signed CA Certificate  
```

Comment:
Can we add a note below title indicating that using this functionality is insecure and should not be exercised in production scenarios?

Reply:
Ok, I'm confused. How do we configure self-signed certs for Prod?

Reply:
@natereid72 the point is that you likely _shouldn't_ be using self-signed certs in a production scenario. There are certainly cases where an organization may opt to use internal certs and this makes sense for them, but they should make sure they know what they are doing (i.e. for the common user, using a custom CA is not going to be a great idea in production).

Reply:
ok, imo, self-signed certs are common in enterprises that run their own certificate infrastructure. It's only insecure if you're making on on your workstation for testing and it's not part of a larger managed infrastructure.

Reply:
> self-signed certs are common in enterprises that run their own certificate infrastructure

Agreed, but if you are not a large enterprise with robust PKI then using self-signed certificates is not a great idea outside of testing scenarios. I think it is worth adding a note to that effect.

Reply:
added

Reply:
nice -- looks great 👍🏻